src/esi/Esi.cc

```diff
@@ -930,7 +930,7 @@ void
 ESIContext::addStackElement (ESIElement::Pointer element)
 {
     /* Put on the stack to allow skipping of 'invalid' markup */
-    Must(parserState.stackdepth < 10);
+    Must2(parserState.stackdepth < 10, "Too many nested elements");
```

Comment:
Can we avoid duplicating this magic `10`context? Giving it a name may also explain why it is not just `STACK_DEPTH_LIMIT`.

Reply:
Well, it is a different stack. But sure, done.